### PR TITLE
Update README.md ... adding additional info ...

### PR DESCRIPTION
**Pull request by aiamuzz**
Would this additional bits of information be helpful ?

### CODE REVIEW

**Comment by plata** [comment]
I think we misunderstood each other. I understood that you wanted to add a README with the compat dependency to https://github.com/flathub/org.phoenicis.playonlinux.

However, we can also update the README here (adding an "installation" section like you did is a good idea). Nevertheless, we should not restrict this to flatpak because there are other possibilities as well (e.g. deb). Therefore, I would suggest to only link to https://phoenicisorg.github.io/phoenicis/Users/installation/ and explain everything there.

**Comment by aiamuzz** [comment]
> I think we misunderstood each other. I understood that you wanted to add a README with the compat dependency to https://github.com/flathub/org.phoenicis.playonlinux.
> 
Yeah i meant only add it as info ... at least the users will know that the compat dependency is needed ... else they will have to report the issue and somebody will have to make it known to them ... 

> However, we can also update the README here (adding an "installation" section like you did is a good idea). Nevertheless, we should not restrict this to flatpak because there are other possibilities as well (e.g. deb). Therefore, I would suggest to only link to https://phoenicisorg.github.io/phoenicis/Users/installation/ and explain everything there.


ok makes sense ... but i am kind of new to github.com ... i see github.io ... and the UI is different there ... how can i create a PR or add information to the Installation page there ?

Honestly the build and installation instructions are pretty confusing both on github.com and github.io ...

**Comment by plata** [comment]
It is generated from: https://github.com/PhoenicisOrg/phoenicis/blob/master/docs/_docs/Users/installation.md

File: README.md [hunk 1]
```diff
@@ -30,8 +30,15 @@ cd phoenicis-javafx/
 mvn exec:java
 ```
 
+For more details on installing on various linux distro's [Install/Build](https://phoenicisorg.github.io/phoenicis/Developers/build/)
```
Comment (AsciiWolf) [inline README.md]: Typo: Should be "distros", not "distro's". Also, "For more details on installing on various linux distros, see:" sounds better.

File: README.md [hunk 2]
```diff
@@ -30,8 +30,15 @@ cd phoenicis-javafx/
...
+
 For more details (e.g. regarding dependencies) consider the [documentation](https://phoenicisorg.github.io/phoenicis/).
 
+## Flatpak Install
```
Comment (plata) [inline README.md]: We can use the official flathub badge for this purpose. See #2145.

**Comment by plata** [comment]
Superseded by #2145 and #2146.

**Pull request closed without merging**